docs/accessibility/Map.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Splitter
```

Comment:
```suggestion
title: Map
```

Reply:
Fixed in https://github.com/telerik/kendo-themes/pull/3214/commits/f1e1d9fb4d1f3e202c270b5446538a6fe84db2f7